wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+SecurityClassification.swift

```diff
@@ -261,4 +261,62 @@ final class ZMUserSessionTests_SecurityClassification: ZMUserSessionTestsBase {
         XCTAssertEqual(classification, .notClassified)
     }
 
+    func testThatItReturnsNotClassified_WhenFeatureIsEnabled_WhenConversationDomainNotClassified() {
+        // given
+        let otherUser1 = createUser(moc: syncMOC, domain: UUID().uuidString)
+        let otherUser2 = createUser(moc: syncMOC, domain: nil)
```

Comment:
I can see, you've added domain for second user in the other test. Why is there a difference here?